shared/templates/template_BASH_shell_lineinfile

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+{{{ bash_shell_file_set(path=PATH, parameter=PARAMETER, value=VALUE, no_quotes=NO_QUOTES) }}}
```

Comment:
I used it as a replacement for our custom code for accounts_tmout. The template generates weird code. Notice the `^Match` regex. It probably should be `^\s*TMOUT` in my example.

```
  1 # platform = multi_platform_all
  2 # reboot = false
  3 # strategy = restrict
  4 # complexity = low
  5 # disruption = low
  6 
  7   
  8   if [ -e "/etc/profile" ] ; then
  9     LC_ALL=C sed -i "/^\s*TMOUT=/d" "/etc/profile"
 10 else
 11     touch "/etc/profile"
 12 fi
 13 cp "/etc/profile" "/etc/profile.bak"
 14 # Insert before the line matching the regex '^Match'.
 15 line_number="$(LC_ALL=C grep -n "^Match" "/etc/profile.bak" | LC_ALL=C sed 's/:.*//g')"
 16 if [ -z "$line_number" ]; then
 17     # There was no match of '^Match', insert at
 18     # the end of the file.
 19     printf '%s\n' "TMOUT='100'" >> "/etc/profile"
 20 else
 21     head -n "$(( line_number - 1 ))" "/etc/profile.bak" > "/etc/profile"
 22     printf '%s\n' "TMOUT='100'" >> "/etc/profile"
 23     tail -n "+$(( line_number ))" "/etc/profile.bak" >> "/etc/profile"
 24 fi
 25 # Clean up after ourselves.
 26 rm "/etc/profile.bak"
```